iryna_bondarenko/models/country.js

```diff
@@ -0,0 +1,20 @@
+var mongoose = require('mongoose');
```

Comment:
The tabbing thing doesn't seem to be fixed.  This is so weird, but I know it's not your fault.  :)